tcrudge/utils/schema.py

```diff
@@ -218,10 +218,10 @@ def _get_properties(self, recurse=True):
         if not recurse:
             return dict(self._properties)
 
-        properties = {}
-        for prop, subschema in self._properties.items():
-            properties[prop] = subschema.to_dict()
-        return properties
+        return {
+            prop: subschema.to_dict()
+            for prop, subschema in self._properties.items()
+        }
```

Comment:
Function `Schema._get_properties` refactored with the following changes:

- Convert for loop into dictionary comprehension ([dict-comprehension](https://docs.sourcery.ai/refactorings/dict-comprehension/))
- Inline variable that is immediately returned ([inline-immediately-returned-variable](https://docs.sourcery.ai/refactorings/inline-immediately-returned-variable/))